lib/galaxy/containers/docker_model.py

```diff
@@ -111,10 +50,6 @@ def __str__(self):
             volume_for_cmd_line = f'"{volume_str}"'
         return volume_for_cmd_line
 
-    def to_native(self):
-        host_path = self.host_path or self.path
-        return (self.path, {host_path: {"bind": self.path, "mode": self.mode}})
-
 
 class DockerContainer(Container):
```

Comment:
This also seems unused? Dropping this would allow us to also remove `Container` and `ContainerPort` classes from `lib/galaxy/containers/__init__.py`